src/main/java/dataStructures/bTree/README.md

```diff
@@ -0,0 +1,104 @@
+# B-Trees
+
+## Background
+Is the fastest way to search for data to store them in an array, sort them and perform binary search? No. <br>
```

Comment:
yup maybe can emphasize that new insertions would require re-sorting as well. Expensive overhead